### DISCUSSION TITLE
应该如何分享自己的规则到远程订阅呢

### DISCUSSION THREAD

**Discussion by huifei123** [gkd-kit/gkd#473] [General]
如题

**Comment by huowutong** [comment]
提交pr，等待review合并，最后官方规则更新，规则就到远程订阅列表中了。
说实话，这个流程有点长，还不如自己在本地订阅中添加规则，然后提交issue，待远程订阅更新之后再删除本地订阅。

**Comment by lisonge** [comment]
如何你想制作自己的远程订阅，你可以把自己的订阅文件提交到 github，然后更改 id 为远程订阅的任意id

然后通过 raw 链接添加订阅
